target_redshift/db_sync.py

```diff
@@ -564,23 +569,17 @@ def load_csv(self, s3_key, count, size_bytes, compression=False, manifest=False)
                     manifest_option = ""
 
                 # Step 4: Load into the stage table
-                copy_sql = """COPY {table} ({columns}) FROM 's3://{s3_bucket}/{s3_key}'
+                columns = ", ".join([c["name"] for c in full_columns])
+                s3_bucket = self.connection_config["s3_bucket"]
+
+                run(
+                    f"""COPY {stage_table} ({columns}) FROM 's3://{s3_bucket}/{s3_key}'
```

Comment:
I converted out of the format style to use inline formatting. It's just so much easier to read imho. lemme know if this is a problem.

Reply:
Agreed, definitely easier to read